references/cli/adopt.go

```diff
@@ -322,7 +328,8 @@ func (opt *AdoptOptions) MultipleRun(f velacmd.Factory, cmd *cobra.Command) erro
 		opt.AppName = r[0].GetName()
 		opt.AppNamespace = r[0].GetNamespace()
 		if err := opt.Run(f, cmd); err != nil {
-			return err
+			_, _ = fmt.Fprintf(opt.Out, "Error: failed to adopt %s/%s: %s", opt.AppNamespace, opt.AppName, err.Error())
```

Comment:
How do you define the log level? What is the different of Warning and Error?

Reply:
Currently we do not define the log level in `out`. Technically, it's more like output but not logs to users.